src/components/auth/AuthForm.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div v-if="username">
+    <div>
+      {{ username }}
+    </div>
+    <ButtonText
+      text="log out"
+      @click="logOut"
+    />
+  </div>
+  <form
+    v-else
+    class="auth-form"
+    @submit.prevent="logIn(logInData)"
+  >
+    <InputCustom
+      label="username"
+      v-model="logInData.username"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="password"
+      type="password"
+      v-model="logInData.password"
+      :required="true"
+      @input="clearMessage"
+    />
+    <div class="message-container">
```

Comment:
don't you think that it's better to move to a component if we already use it in two places?